deploy/operator/10_role_binding.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
```

Comment:
RoleBinding

Reply:
ditto: cannot be RoleBinding

Reply:
you are right, I've copied that comment without thinking too much